ProblemSolving/Greedy/minimum-absolute-difference-in-an-array.js

```diff
@@ -0,0 +1,16 @@
+// https://www.hackerrank.com/challenges/minimum-absolute-difference-in-an-array/problem
+
+const getMinDifferenceInSorted = sorted => {
+  let minDiff = Infinity;
+  sorted.forEach((value, index, arr) => {
```

Comment:
한줄로 표현할 수 있는 코드를 함수로 표현하는 건 그냥 연습하는 건가요?
```suggestion
for (let i = 0; i < arr.length - 1; i++) {
        min = (min > Math.abs(arr[i] - arr[i + 1])) ? Math.abs(arr[i] - arr[i + 1]) : min;
    }
```

Reply:
왜 for문 대신 forEach 를 사용했는지를 묻는건가요 ?